models/ed/R/write.configs.ed.R

```diff
@@ -52,14 +54,17 @@ convert.samples.ED <- function(trait.samples) {
   if ("root_respiration_rate" %in% names(trait.samples)) {
     rrr1 <- as.numeric(trait.samples[["root_respiration_rate"]])
     rrr2 <- rrr1 * DEFAULT.MAINTENANCE.RESPIRATION
-    trait.samples[["root_respiration_rate"]] <- PEcAn.utils::arrhenius.scaling(rrr2, old.temp = 25, new.temp = 15)
+    trait.samples[["root_respiration_rate"]] <-
+      PEcAn.utils::arrhenius.scaling(rrr2, old.temp = 25, new.temp = 15)
     # model version compatibility (rrr and rrf are the same)
-    trait.samples[["root_respiration_factor"]] <- trait.samples[["root_respiration_rate"]]
+    trait.samples[["root_respiration_factor"]] <-
+      trait.samples[["root_respiration_rate"]]
```

Comment:
There's a lot of unnecessary line formatting changes in this PR. Some might be improvements, some are neutral, but some (like this one and the one above) clearly make the code less readable.

Reply:
ok, I reverted the line breaks for long lines